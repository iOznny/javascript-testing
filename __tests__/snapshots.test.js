const client = {
    name: 'Demos',
    balance: 500,
    type: 'Premium'
}

describe('Testing Client', () => {
    // Testing
    test('Is Demo', () => {
        expect(client.name).toBe('Demo');
    });

    // Snapshot
    test('Testing with Snapshot', () => {
        // Create snapshot
        expect(client).toMatchSnapshot();        
    });
}); 

// Update snapshot in console command:
// npm test -- -u