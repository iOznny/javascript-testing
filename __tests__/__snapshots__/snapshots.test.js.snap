// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Client Testing with Snapshot 1`] = `
Object {
  "balance": 500,
  "name": "Demo",
  "type": "Premium",
}
`;
